Sources/Classes/RSEventRepository.m

```diff
@@ -15,6 +15,10 @@
 #import "UIViewController+RSScreen.h"
 
 static RSEventRepository* _instance;
+RSDBMessage* _dbMessage;
+NSLock* lock;
+dispatch_source_t source;
+dispatch_queue_t queue;
```

Comment:
Move all this to new variable to header file.

Reply:
Okay, Sure

Reply:
Addressed now